app/static/styles/main.css

```diff
@@ -95,6 +97,19 @@ body {
     font-weight: 700;
 }
 
+.about-us h2 {
+    margin-right: 10px;
+    margin-left: 15px;
+    color: #1C539F;
+    font-size: 40px;
```

Comment:
As @Pradyuman7 suggested, consider using the `em` unit for specifying font sizes as it offers a more flexible and scalable approach compared to fixed units like pixels.